.azure-pipelines/azure-pipelines.yml

```diff
@@ -1,4 +1,4 @@
-# CI and PR build script
+# CI and PR build scriptss
```

Comment:
typo: scripts

Reply:
This is a dummy PR @anuragc617 which I created for testing the stalled check runs. Please ignore this PR.